packages/xstate-wallet/src/workflows/application.ts

```diff
@@ -222,13 +231,15 @@ const mockServices = {
   invokeJoinMachine: () => {},
   invokeClosingMachine: () => {}
 };
-const mockActions = {
-  sendToOpponent: () => {},
-  updateStore: () => {},
-  sendChannelUpdatedNotification: () => {},
-  hideUi: () => {},
-  displayUi: () => {},
-  assignChannelId: () => {}
+const mockActions: Actions = {
+  sendToOpponent: 'sendToOpponent',
+  updateStore: 'updateStore',
+  sendChannelUpdatedNotification: 'sendChannelUpdatedNotification',
+  hideUi: 'hideUi',
+  displayUi: 'displayUi',
+  assignChannelId: 'assignChannelId'
 };
+
+export const config = generateConfig(mockActions);
```

Comment:
Why do we need this? It has the same name as another `const` above (not exported), and does not appear to be imported anywhere.
```suggestion
```

Reply:
We need this so that `yarn generateConfigs` will work. It isn't ideal, but that's how it works for now.